features/bevragen/persoon-beperkt/verblijfplaats/dev/autorisatie-gba.feature

```diff
@@ -0,0 +1,181 @@
+      | instance | /haalcentraal/api/brp/personen                                          |
+
+    @fout-case
+    Scenario: Afnemer vraagt om groep verblijfplaats en is niet geautoriseerd voor functie adres
+      Gegeven de afnemer met indicatie '12345' heeft de volgende 'autorisatie' gegevens
+      | Rubrieknummer ad hoc (35.95.60)                                         | Medium ad hoc (35.95.67) | Datum ingang (35.99.98) |
+      | 10120 80910 81110 81115 81120 81130 81140 81150 81160 81170 81210 81310 | N                        | 20201128                |
+      En de geauthenticeerde consumer heeft de volgende 'claim' gegevens
+      | naam         | waarde |
+      | afnemerID    | 12345  |
+      | gemeenteCode | 0518   |
+      Als gba personen wordt gezocht met de volgende parameters
+      | naam       | waarde                             |
+      | type       | ZoekMetPostcodeEnHuisnummer        |
+      | postcode   | 2497BV                             |
+      | huisnummer | 103                                |
+      | fields     | burgerservicenummer,verblijfplaats |
+      Dan heeft de response een object met de volgende gegevens
+      | naam     | waarde                                                                  |
+      | type     | https://datatracker.ietf.org/doc/html/rfc7231#section-6.5.3             |
+      | title    | U bent niet geautoriseerd voor één of meerdere opgegeven field waarden. |
+      | status   | 403                                                                     |
+      | detail   | De foutieve fields waarden zijn: fields[1].                             |
+      | code     | authorization                                                           |
+      | instance | /haalcentraal/api/brp/personen                                          |
+
+  Rule: Een gemeente als afnemer is geautoriseerd voor alle verblijfplaatsgegevens van eigen inwoners
+
+    Scenario: Gemeente vraagt om groep verblijfplaats van een eigen inwoner en functie adres (81010) zit niet in de autorisatie
+      Gegeven de afnemer met indicatie '12345' heeft de volgende 'autorisatie' gegevens
+      | Rubrieknummer ad hoc (35.95.60)     | Medium ad hoc (35.95.67) | Datum ingang (35.99.98) |
+      | 10120 81110 81120 81160 81210 81310 | N                        | 20201128                |
+      En de geauthenticeerde consumer heeft de volgende 'claim' gegevens
+      | naam         | waarde |
+      | afnemerID    | 12345  |
+      | gemeenteCode | 0599   |
+      Als gba personen wordt gezocht met de volgende parameters
+      | naam                | waarde                             |
+      | type                | RaadpleegMetBurgerservicenummer    |
+      | burgerservicenummer | 000000024                          |
+      | fields              | burgerservicenummer,verblijfplaats |
+      Dan heeft de response een persoon met de volgende 'verblijfplaats' gegevens
```

Comment:
Waarom wordt hier alleen maar straat en functieAdres geleverd. Moet 'huisletter', 'huisnummertoevoeging', 'postcode', 'woonplaats' en 'datumAanvangAdreshouding' niet ook geleverd worden?

Reply:
dat laatste is precies wat er niet goed was. gewijzigd naar ZoekMetPostcodeEnHuisnummer. Dan klopt de Dan stap wel. Straat wordt geleverd om het type te bepalen, functieAdres wordt geleverd omdat dit het enige veld is binnen verblijfplaats van PersoonBeperkt.